pkg/cmd/inspect/inspect.go

```diff
@@ -0,0 +1,55 @@
+package inspect
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"kitops/pkg/lib/constants"
+	"kitops/pkg/lib/repo"
+	"kitops/pkg/output"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"oras.land/oras-go/v2/errdef"
+)
+
+func inspectReference(ctx context.Context, opts *inspectOptions) (*ocispec.Manifest, error) {
+	if opts.modelRef.Registry == repo.DefaultRegistry {
+		// Local only check
+		return getLocalManifest(ctx, opts)
+	}
+	if opts.checkRemote {
+		// Remote only check
+		return getRemoteManifest(ctx, opts)
+	}
+
+	// Check locally first; if not found check remote
+	manifest, err := getLocalManifest(ctx, opts)
```

Comment:
same comment as info. What happens if the local and remote modelkits that refer to the same tag are not a match?